Adiciona post retrospectiva + visus analógicas (#31)
OpenDevUFCG/OpenDevUFCGBlog: texts/v0.0.1/POST_RETROSPECTIVA_VISU_ANALOGICAS.md

Olá! Chegou o final do ano, para muitos é tempo de férias, se reunir com a família e de relembrar momentos do ano que está chegando ao fim. A Opendev também não ia deixar passar em branco isso, então, que tal recaptular alguns acontecimentos do ano através de visualizações analógicas?

### Antes de mostrar as visus, um questionamento, o que são visualizações analógicas? 

São desenhos ou ilustrações, feitas a mão mesmo, sobre algum dado. A referência dessas visus, é o trabalho [Dear data](http://www.dear-data.com/theproject) das designers Giorgia Lupi e Stefanie Posavec, em 2015, quando decidiram criar visualizações sobre dados do dia a dia e enviá-las através de cartões postais como forma de comunicar uma a outra o que estava acontecendo em suas rotinas. Por exemplo:

![Exemplo 1](https://thepracticaldev.s3.amazonaws.com/i/e6soc2j62svs4xmzx8id.jpeg)

![Exemplo 2](https://thepracticaldev.s3.amazonaws.com/i/fvh56hxw83setj9828ip.jpg)

As imagens acima foram tiradas do livro Dear Data e, ao final do post, indicarei algumas referências do trabalho delas.

### Então, sabendo um pouco sobre visus analógicas, bora ver como os dados da Opendev se comportaram ao longo do ano de 2019? 

> O ano 2019, a organização fez 1 ano, e como resultado, tivemos crescimento da organização, com a criação de novos projetos, como podemos observar na imagem a seguir.

Reviewer: Fiz umas sugestões pra diminuir umas repetições na frase, mas fique à vontade para mudar caso eu tenha mudado a ideia/você prefira de outra forma \o/
```suggestion
Em 2019, a OpenDevUFCG completou um ano e, como resultado, tivemos o crescimento da equipe e do número de projetos, como podemos observar na imagem a seguir.
```

![Integrantes](https://thepracticaldev.s3.amazonaws.com/i/i7gekvfk1uh8u1a1qmda.jpeg) 

> Tendo em vista que o core cresceu, também tivemos mais ideias de projetos surgindo, é o que você vai ver na próxima imagem:

Reviewer: ```suggestion
Tendo em vista que o time _Core_ cresceu, também tivemos mais ideias de projetos surgindo. É o que você vai ver na próxima imagem:
```

![Projetos](https://thepracticaldev.s3.amazonaws.com/i/s69p68sabq47onmwc4ji.jpeg)

> É de grande satisfação ver o crescimento de nossos projetos e que nosso espírito de comunidade está impactando muitas pessoas, e como resultado, ver que nossa ideia está influenciando as pessoas a saber mais e participar de projetos open source. 

Reviewer: ```suggestion
É de grande satisfação ver o crescimento de nossos projetos e que nosso espírito de comunidade está impactando muitas pessoas, e como resultado, ver que nossa ideia está influenciando as pessoas a querer saber mais e participar de projetos _open source_. 
```

> Então movidos por isso, pensamos em organizar o Hacktoberfest, um evento para reunir boas ideias e pessoas legais para ajudar à comunidade, através de palestras, minicursos, hacktime e chuva de ideias. E, agradecemos muito pelo apoio de todos, doadores e patrocinadores que compraram a ideia e deram apoio para o evento acontecer. Mas, deixando um pouco as palavras de lado, que tal ver os dados disso?

Reviewer: ```suggestion
Então, movidos por isso, pensamos em organizar o **Hacktoberfest OpenDevUFCG**, um evento para reunir boas ideias e pessoas legais para ajudar a comunidade através de atividades como palestras, minicursos, HackTime e abertura para sugestões. Agradecemos muito pelo apoio de todos, doadores e patrocinadores que compraram a ideia e deram apoio para o evento acontecer! Mas, deixando um pouco as palavras de lado, que tal ver os dados disso tudo?
```

![Hacktoberfest](https://thepracticaldev.s3.amazonaws.com/i/irk9d9e5jedlamja2a21.jpeg)

Bem, tentei destacar os principais pontos da nossa organização esse ano. E, como prometido, links com mais detalhes sobre o projeto Dear Data:

> [O projeto](http://www.dear-data.com/theproject)

Reviewer: ```suggestion
- [O projeto](http://www.dear-data.com/theproject);
```

> [Vídeo TED](https://www.ted.com/talks/giorgia_lupi_how_we_can_find_ourselves_in_data)

Reviewer: ```suggestion
- [Vídeo TED](https://www.ted.com/talks/giorgia_lupi_how_we_can_find_ourselves_in_data);
```

> [Livro](http://www.dear-data.com/thebook)

Reviewer: ```suggestion
- [Livro](http://www.dear-data.com/thebook);
```

> [Medium @Giorgia Lupi](https://medium.com/@giorgialupi)

Reviewer: ```suggestion
- [Medium @Giorgia Lupi](https://medium.com/@giorgialupi).
```

> Então gente, é isto, de novo gostaria de agradecer a cada um que acredita em nossos projetos e na nossa comunidade, foi um ano muito legal para nossa organização, de bastante crescimento e de aprendizado, e sempre saibam, estamos dispostos para ajudar. Obrigada pela leitura, até ano que vem!

Reviewer: ```suggestion
Então, gente, é isto, de novo gostaria de agradecer a cada um que acredita em nossos projetos e na nossa comunidade! Foi um ano muito legal para nossa organização, de bastante crescimento e de aprendizado, e sempre saibam, estamos dispostos para ajudar. Obrigada pela leitura, até ano que vem!
```
